tools/common_lib/src/tensor_shape.h

```diff
@@ -46,6 +46,33 @@ struct TensorShape
 
         return ret;
     }
+
+    inline std::uint32_t get_element_count() const
+    {
+        std::uint32_t ret = 0;
```

Comment:
Function could be much simplified, by i.e.:
1) Initialize `ret` with `n`
2) create lambda functions and reuse it for c, h, w dimensions.

Reply:
Done